src/static/css/infusion-docs.css

```diff
@@ -58,6 +58,16 @@ pre {
     white-space: pre-wrap;
 }
 
+a:focus {
+    outline: solid 0.2rem #008cba;
```

Comment:
In Chrome and Safari, this looks a little bit odd: there are gaps in the border. Any chance you can figure out what's going on?